src/Common/src/CoreLib/System/Text/ValueStringBuilder.cs

```diff
@@ -185,7 +182,7 @@ private void Grow(int requiredAdditionalCapacity)
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private void Clear()
+        public void Dispose()
```

Comment:
Renamed Clear to Dispose while I was on it to be consistent with other builder helper types.